service/grid/getData.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+import {
+    getMappedCellValues,
+    getMappedRows,
+    getMappedColumns,
+    getSortedColumnsByIDs,
+} from '~/model/mappers/gridDataMapper';
+import { COLUMNS_IDS } from '~/defaults/grid/cookies';
+
+export const getGridData = ($axios, $cookies, path, params) => $axios
+    .$get(path, { params })
+    .then(({
+        collection,
+        columns,
+        info: { count, filtered },
+    }) => {
+        const sortedColumns = params.columns ? getSortedColumnsByIDs([...columns], params.columns, 'id') : columns;
```

Comment:
Do you need to use a spread operator on the `columns` variable ? You copy array ?